dasharo-security/tpm-support.robot

```diff
@@ -146,3 +147,50 @@ TPM003.003 Check TPM Physical Presence Interface (Windows 11)
 #    Skip If    not ${tpm_support}    TPM003.004 not supported
 #    Skip If    not ${tests_in_ubuntu_support}    TPM003.004 not supported
 # TODO: https://docs.dasharo.com/unified-test-documentation/dasharo-security/200-tpm-support/#tpm003004-change-active-pcr-banks-with-tpm-ppi-firmware
+
+TPM004.001 Check TPM Clear procedure
+    [Documentation]    This test aims to verify whether the TPM Clear procedure
+    ...    works properly
+    Skip If    not ${TPM_SUPPORT}    TPM004.001 not supported
+    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    TPM004.001 not supported
+
+    Power On
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    Detect Or Install Package    tpm2-tools
+
+    # Take ownership
+    Execute Linux Tpm2 Tools Command    tpm2_changeauth --quiet -c owner pass
+    Execute Linux Tpm2 Tools Command    tpm2_changeauth --quiet -c lockout pass
+    Execute Linux Command    tpm2_createprimary -Q --hierarchy=o --key-context=/tmp/test --key-auth=pass2 -P pass
+    Execute Linux Tpm2 Tools Command    tpm2_evictcontrol -Q -C o -P pass -c /tmp/test 0x81000001
+    Execute Linux Command    rm /tmp/test
+
+    # Check ownership
+    Execute Linux Command    ! tpm2_changeauth --quiet -c owner 2>/dev/null
+    ${out}=    Execute Linux Command    echo $?
+    Should Be Equal    ${out}    0\n
+
+    # Clear TPM
+    Power On
+    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
```

Comment:
@PLangowski we could use here keywords created by @m-iwanicki https://github.com/Dasharo/open-source-firmware-validation/blob/encrypted-rootfs-release/lib/tpm-support-lib.robot#L10, maybe even add more run Clear TPM procedure

Reply:
New KW added for that https://github.com/Dasharo/open-source-firmware-validation/pull/184/files#diff-c1e2cb8ca8f6eb20d6bfb97aceffb3b83887bf042ac54f7c82c5b4fc833a3cf6R56